.github/workflows/ci-build.yaml

```diff
@@ -33,6 +33,18 @@ jobs:
         if: github.ref == 'refs/heads/master'
         run: bash <(curl -s https://codecov.io/bash)
 
+  tests-windows:
+    name: Windows Unit Tests
+    runs-on: windows-2022
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v4
+        with:
+          go-version: "1.21"
+          cache: true
+      - run: bash -c 'make test STATIC_FILES=false GOTEST="go test -p 20"'
```

Comment:
We should run coverage here as well; the coverage reporter typically takes the union of multiple reports from a single CI run in order to handle scenarios like this one (some different code paths for different OSes in this case)

Reply:
👍 I wasn't sure about it and that's why I removed it. Will re-add. Thanks